crates/modelardb_server/src/query/mod.rs

```diff
@@ -158,8 +195,14 @@ impl ModelTable {
             infinite_source: false,
         };
 
-        let physical_parquet_predicates =
-            convert_expr_to_physical_expr(parquet_predicates, COMPRESSED_SCHEMA.0.clone())?;
+        let physical_parquet_predicates = if let Some(parquet_predicates) = parquet_predicates {
```

Comment:
Fixed.

Reply:
I think the change you made makes sense but i was originally referring to this `parquet_predicates`:
`if let Some(parquet_predicates) = **parquet_predicates**` 

Changing that to `maybe_parquet_predicates` would make sense since it would avoid having the `Option` have the same name as the unwrapped `parquet_predicates` inside `Some`. 

In general it might be best if we try to be consistent with using `maybe_` as a prefix for `Option` variables.

Reply:
Fixed.